src/odemis/driver/test/orsay_test.py

```diff
@@ -1269,67 +1284,78 @@ def test_gunOn(self):
     def test_lifetime(self):
         """Check that the lifetime VA is updated correctly"""
         connector_test(self, self.fib_source.lifetime, self.fib_source._hvps.SourceLifeTime,
-                       [(0.1, 0.1), (0.2, 0.2)], readonly=True)
+                       [(0.1, 0.1), (0.2, 0.2)], hw_safe=False, readonly=True)
 
     def test_currentRegulation(self):
         """Check that the currentRegulation VA is updated correctly"""
         connector_test(self, self.fib_source.currentRegulation, self.fib_source._hvps.BeamCurrent_Enabled,
-                       [(True, "True"), (False, "False")], readonly=True, settletime=1)
+                       [(True, "True"), (False, "False")], hw_safe=False, readonly=True, settletime=1)
 
     def test_sourceCurrent(self):
         """Check that the sourceCurrent VA is updated correctly"""
         connector_test(self, self.fib_source.sourceCurrent, self.fib_source._hvps.BeamCurrent,
-                       [(1e-5, 1e-5), (0, 0)], readonly=True)
+                       [(1e-6, 1e-6), (2e-6, 2e-6)], hw_safe=False, readonly=True, settletime=30)
 
+    # TODO K.K. test again
     def test_suppressorVoltage(self):
         """Check that the suppressorVoltage VA is updated correctly"""
-        connector_test(self, self.fib_source.suppressorVoltage, self.fib_source._hvps.Suppressor,
-                       [(10, 10), (0, 0)], hw_safe=True, settletime=1)
-
-    def test_heaterCurrent(self):
-        """Check that the heaterCurrent VA is updated correctly"""
-        connector_test(self, self.fib_source.heaterCurrent, self.fib_source._hvps.Heater,
-                       [(1, 1), (0, 0)], hw_safe=True, settletime=1)
-
-    def test_heater(self):
-        """Check that the heater VA is updated correctly"""
-        connector_test(self, self.fib_source.heater, self.fib_source._hvps.HeaterState,
-                       [(True, orsay.HEATER_ON), (False, orsay.HEATER_OFF)],
-                       hw_safe=True, settletime=1)
-
-        if TEST_NOHW == "sim":  # This part of the test is only safe in simulation
-            self.fib_source._hvps.HeaterState.Target = orsay.HEATER_RISING
-            sleep(0.5)
-            self.assertTrue(self.fib_source.heater.value)
-
-            self.fib_source._hvps.HeaterState.Target = orsay.HEATER_ERROR
-            sleep(0.5)
-            self.assertFalse(self.fib_source.heater.value)
-            self.assertIsInstance(self.fib_source.state.value, HwError)
-            self.assertIn("FIB source forced to shut down", str(self.fib_source.state.value))
+        if not TEST_NOHW == 1:
```

Comment:
Shouldn't it be `if TEST_NOHW == "sim":` ? The comment just below seems to indicate you want to skip when in simulator mode.